com.fortress.weblauncher.json

```diff
@@ -0,0 +1,81 @@
+{
+    "app-id" : "com.fortress.weblauncher",
```

Comment:
>  I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).

Given what's at `fortress.com` I really ask the question. Is this domain under the control of the developer.

Please do pick random domains for your app-id